packages/lib/src/visually-hidden/index.js

```diff
@@ -44,8 +44,11 @@ function visuallyHidden({
   );
 }
 
-visuallyHidden.focusable = function focusable({ important = true } = {}) {
+function focusable({ important = true } = {}) {
   return visuallyHidden({ important, focusable: true });
-};
+}
+// TODO: @Steve, do we need to append this to the function?
+visuallyHidden.focusable = focusable;
```

Comment:
given how we export from `lib`, i think we do

i don't think that the `export { focusable }` added below will allow one to do `import { visuallyHidden } from '@govuk-react/lib';` and then use `visuallyHidden.focusable`

Reply:
ok, have removed comment